src/commands/Management/Configuration/manageReactionRoles.ts

```diff
@@ -0,0 +1,137 @@
+import { chunk } from '@klasa/utils';
+import { DbSet } from '@lib/structures/DbSet';
+import { SkyraCommand, SkyraCommandOptions } from '@lib/structures/SkyraCommand';
+import { UserRichDisplay } from '@lib/structures/UserRichDisplay';
+import { PermissionLevels } from '@lib/types/Enums';
+import { GuildSettings, ReactionRole } from '@lib/types/settings/GuildSettings';
+import { ApplyOptions, CreateResolvers } from '@skyra/decorators';
+import { BrandingColors } from '@utils/constants';
+import { LongLivingReactionCollector } from '@utils/LongLivingReactionCollector';
+import { displayEmoji, resolveEmoji } from '@utils/util';
+import { Guild, MessageEmbed, Role, TextChannel } from 'discord.js';
+import { ArrayActions, KlasaMessage } from 'klasa';
+
+@ApplyOptions<SkyraCommandOptions>({
+	aliases: ['mrr'],
```

Comment:
```suggestion
	aliases: ['mrr', 'managereactionrole', 'managerolereaction', 'managerolereactions'],
```

singular is always welcome next to plural and I'm pretty sure autoaliases doesn't do that. Also added the previous names for people who will type that.